README.md

```diff
@@ -88,7 +88,7 @@ Required keys can be obtained from [https://www.instagram.com/developer/authenti
 
 #### Google Maps
 
-Google maps is used to get information about location (info about country, city, latitude and longitude).
+Google maps are used to get information about location (info about country, city, latitude and longitude).
```

Comment:
In this cases the application "Google maps" is a singular application, there is only one of it.